# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mp4parse/link-u-avif-sample-images/scripts/compare.sh
# Commit: a651d4323066
# Full Hash: a651d43230664837c308663382ccc3bcd7298dc2
# Author: zaggy1024 <Zaggy1024@gmail.com>
# Date: 2023-01-28 21:11:06
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Bug 1788119 - Part 4 - Update mp4parse-rust for AVIS support. r=kinetik,glandium,supply-chain-reviewers
#   
#   Depends on D156652
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156653
# ==============================================================================

diff -r be008ddb2418 -r a651d4323066 third_party/rust/mp4parse/link-u-avif-sample-images/scripts/compare.sh
--- a/third_party/rust/mp4parse/link-u-avif-sample-images/scripts/compare.sh	Fri Jan 27 21:25:50 2023 +0000
+++ b/third_party/rust/mp4parse/link-u-avif-sample-images/scripts/compare.sh	Fri Jan 27 21:25:51 2023 +0000
@@ -1,23 +1,23 @@
-##!/usr/bin/env bash
-
-avif=$2
-decoded=$3
-
-orig=$(cat Makefile | grep "^${avif}" | sed "s/^${avif}: \(.*\)$/\1/")
-
-if (echo ${avif} | grep "monochrome"); then
-  # FIMXE(ledyba-z): compare monochrome images.
-  score="100.0"
-elif (echo ${avif} | grep "\(rotate\|mirror\|crop\)"); then
-  # FIMXE(ledyba-z): compare transformed images
-  score="100.0"
-else
-  score=$(compare -metric PSNR ${orig} ${decoded} NULL: 2>&1 || true)
-fi
-if test $(echo "${score} >= 35.0" | bc -l) -eq 1; then
-  echo "Passing: ${decoded}: ${score}"
-  exit 0
-else
-  echo "Failed: ${decoded}: ${score} (vs ${orig})"
-  exit -1
-fi
+##!/usr/bin/env bash
+
+avif=$2
+decoded=$3
+
+orig=$(cat Makefile | grep "^${avif}" | sed "s/^${avif}: \(.*\)$/\1/")
+
+if (echo ${avif} | grep "monochrome"); then
+  # FIMXE(ledyba-z): compare monochrome images.
+  score="100.0"
+elif (echo ${avif} | grep "\(rotate\|mirror\|crop\)"); then
+  # FIMXE(ledyba-z): compare transformed images
+  score="100.0"
+else
+  score=$(compare -metric PSNR ${orig} ${decoded} NULL: 2>&1 || true)
+fi
+if test $(echo "${score} >= 35.0" | bc -l) -eq 1; then
+  echo "Passing: ${decoded}: ${score}"
+  exit 0
+else
+  echo "Failed: ${decoded}: ${score} (vs ${orig})"
+  exit -1
+fi